src/pages/index.js

```diff
@@ -18,6 +18,7 @@ import stichImage from '../images/2x/Stitch-triggers@2x.png';
 import greenPatternImage from '../images/1x/pattern-green.png';
 import meetupsImage from '../images/1x/Meetups.png';
 import buildImage from '../images/1x/Build.png';
+import buildImage3x from '../images/3x/Build@3x.png';
```

Comment:
can we use 2x? Maybe it wasn't good enough?

Reply:
We do have one, I'll use that.